storage/migrations/mysql/1_init.sql

```diff
@@ -238,6 +238,18 @@ CREATE TABLE IF NOT EXISTS `sends` (
   INDEX id_created_at (`id`, `created_at`)
 ) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
 
+CREATE TABLE IF NOT EXISTS `subscriber_events`
+(
+    `id`               VARBINARY(27) PRIMARY KEY NOT NULL,
+    `user_id`          INTEGER UNSIGNED NOT NULL,
+    `subscriber_email` VARCHAR(191) NOT NULL,
+    `event_type`       VARCHAR(50) NOT NULL,
```

Comment:
It would be good if we add a second index for the event type, since we'll be doing grouping by this column.